From 1d7e95e9dd1ad1e393f7620fd478e3db7a9db2e7 Mon Sep 17 00:00:00 2001
From: Mervin Fansler <mervin.fansler@bric.ku.dk>
Date: Wed, 31 Jul 2024 14:17:35 +0200
Subject: [PATCH] skip version record

---
 src/Makevars | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/src/Makevars b/src/Makevars
index 9b7899e..eaf2904 100644
--- a/src/Makevars
+++ b/src/Makevars
@@ -1,9 +1,3 @@
 PKG_CPPFLAGS = -DTMBAD_FRAMEWORK
 PKG_LIBS = $(SHLIB_OPENMP_CXXFLAGS)
 PKG_CXXFLAGS=$(SHLIB_OPENMP_CXXFLAGS)
-
-all: $(SHLIB) pkgver
-
-pkgver:
-	"$(R_HOME)/bin$(R_ARCH_BIN)/Rscript" -e "cat(as.character(packageVersion('TMB')), '\n', sep="", file='../inst/TMB-version')"
-	"$(R_HOME)/bin$(R_ARCH_BIN)/Rscript" -e "cat(as.character(packageVersion('Matrix')), '\n', sep="", file='../inst/Matrix-version')"
